templates/SilverStripe/ElementalBannerBlock/Block/BannerBlock.ss

```diff
@@ -1,27 +1,32 @@
-<%-- Display the image (File) --%>
-<% if $File %>
-    <span class="banner-element__image">
-        $File
-    </span>
-
-<% end_if %>
-<div class="banner-element__content">
-    <% if $Title && $ShowTitle %>
-        <h2 class="banner-element__title">$Title</h2>
-    <% end_if %>
+<a class="banner-block"
+  <% if $CallToActionLink.Page.Link %>
+  <% with $CallToActionLink %>
+  href="{$Page.Link}"
+  <% if $TargetBlank %>target="_blank"<% end_if %>
+  <% if $Description %>title="{$Description.ATT}"<% end_if %>
+  <% end_with %>
+  <% end_if %>
+>
+<span class="banner-block__flex-box" role="banner">
+    <span class="banner-block__content">
+        <% if $Title && $ShowTitle %>
+            <span role="heading" class="banner-block__title h4">$Title</span>
+        <% end_if %>
+        <p>$Content.Summary(30)</p>
+        <% if $CallToActionLink.Text %>
+        <span class="banner-block__call-to-action">
+            {$CallToActionLink.Text.XML}
+            <img class="banner-block__call-to-action-icon" src="$resourceURL('themes/bambusa/images/icon-arrow.svg')" alt="" />
+        </span>
+        <% end_if %>
 
-    $Content
-
-    <%-- Add a CallToActionLink if available --%>
-    <% if $CallToActionLink.Page.Link %>
-        <div class="banner-element__call-to-action-container">
-        <% with $CallToActionLink %>
-            <a href="{$Page.Link}" class="btn btn-primary"
-                <% if $TargetBlank %>target="_blank"<% end_if %>
-                <% if $Description %>title="{$Description.ATT}"<% end_if %>>
-                {$Text.XML}
-            </a>
-        <% end_with %>
-        </div>
+    </span>
+    <% if $File %>
+    <% with $File %>
+    <span class="banner-block__image" role="img" aria-label="$Title">
```

Comment:
I don't think the `role` and `$Title` are appropriate here, you already have a title available to the screen reader as part of the standard text. Either actually set the image title (from `File` metadata?), or leave out the role

Reply:
The $Title here will be the title of the file because of the `<% with %>` statement.

Reply:
Oooh, yep!